### DISCUSSION TITLE
Request to Add AlphaDroid

### DISCUSSION THREAD

**Discussion by helliscloser** [ROM-builders/temporary#18633] [General]
Hello guys, 

Could you please kindly add alphadroid to the supported roms? It's based on crdroid and the source is here 

https://github.com/AlphaDroid-Project

Thank you for the time

**Comment by helliscloser** [comment]
Hello

**Comment by Apon77** [comment]
not popular or not active rom is not added, it is a very new rom, so it won't be added soon, need time.

**Reply by helliscloser** [reply]
Understood. Thanks for your time and for this free service you do.
